lib/src/cmaf/manifest-samples/dash-samples/sample-1/manifest-sample-1.mpd

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Current model is not saving encoding of characters. 
Seems correctly assumed that we are accepting encoding 'utf-8' for this and also because of the way we parse xml.

Reply:
Correct, let's assume for now that we always use the same xml version and the same encoding (utf-8).